.github/workflows/ci.yml

```diff
@@ -103,6 +103,52 @@ jobs:
         run: npm run js-api-spec -- --sassSassRepo ../language --sassPackage ../build/npm
         working-directory: sass-spec
 
+  sass_spec_js_browser:
+    name: "JS API Tests | Dart ${{ matrix.dart_channel }} | Browser"
+
+    strategy:
+      matrix:
+        dart-channel: [stable]
+      fail-fast: false
+
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: browser-actions/setup-chrome@v1
+      - uses: dart-lang/setup-dart@v1.2
+        with:
+          sdk: ${{ matrix.dart-channel }}
+      - run: dart pub get
+
+      - name: Check out sass-spec
+        uses: sass/clone-linked-repo@v1
+        # TODO update this to use "sass/sass-spec" directly
```

Comment:
Remember to update this before merging.

Reply:
If you send a PR to the official sass-spec repo, you can link to it in the PR description here and this PR will automatically run against that one.

Reply:
Fixed.